cypress/integration/test_language.js

```diff
@@ -0,0 +1,19 @@
+describe("Test langugae setting in local storage", function () {
+    it("check english language storage", function () {
+        cy.visit("http://localhost:8080/");

```

Comment:
This is not going to work in a CI (i.e. Jenkins) setting

Reply:
Oh, yes I understand.